app/src/main/java/com/google/android/samples/socialite/ui/chat/ChatViewModel.kt

```diff
@@ -33,6 +34,7 @@ import kotlinx.coroutines.launch
 @HiltViewModel
 class ChatViewModel @Inject constructor(
     private val repository: ChatRepository,
+    private val widgetRepository: WidgetModelRepository,
```

Comment:
Doesn't look like it's used in this class?

Reply:
Fixed